_overviews/scala3-macros/tutorial/reflection.md

```diff
@@ -40,14 +40,14 @@ This will import all the types and modules (with extension methods) of the API.
 
 ## How to navigate the API
 
-The full imported API can be found in the [API documentation for `scala.quoted.Quotes.reflectModule`][reflection doc].
+The full API can be found in the [API documentation for `scala.quoted.Quotes.reflectModule`][reflection doc].
 Unfortunately, at this stage, this automatically generated documentation is not very easy to navigate.
 
 The most important element on the page is the hierarchy tree which provides a synthetic overview of the subtyping relationships of
 the types in the API. For each type `Foo` in the tree:
```

Comment:
It might not be fully clear that `Foo` is a metavariable, we could also use something like `<TYPE>Methods`. But if you think that is not necessary, then I am also ok with how it is.

Reply:
But this can also be addressed in a separate PR. To not block any longer, I am going to accept and merge it as is.